# Definition of ROS service LocSwitchMapSrv for sick localization
# Activates or deactivates rolling buffer log file recording on error report.
# Example call (ROS1):
# rosservice call LocSwitchMapSrv "{active: 1}"
# Example call (ROS2):
# ros2 service call LocSwitchMapSrv sick_lidar_localization/srv/LocSwitchMapSrv "{active: 1}"
# 

# 
# Request (input)
# 

string submapname # The target sub map name inside the map folder. The path should be equal to the one defined in the transition file. Max length = 255 

---

# 
# Response (output)
# 

bool success # true: success response received from localization controller, false: service failed (timeout or error status from controller)
